### ISSUE TITLE
After compression volume control?

### ISSUE THREAD

**Issue by 3941** [vatara/firefox-audio-compressor#11]
Would be nice to get what the title says.
There's boost but it only goes above 100%.
I wouldn't mind some reduction, too.

FrankerfaceZ's compressor, which is a plugin to improve Twitch, implemented it after I suggested it a few years ago, and it now has two volume sliders on the video controls. Not really asking for that, but just allow "negative boost" in your plugin interface.
The volume control on Youtube just turns into pre-gain when the compressor is activated.

So, basically, just turn boost into volume, and allow going lower than 0. Just set the default to 100% or whatever.

**Comment by vatara** [comment]
Sure, that's an easy enough change. I posted an update with this. The behavior stays the same, zero boost keeps the same output volume, but now the range goes from -1 to 1.

**Comment by 3941** [comment]
Awesome! Thank you very much!

**Comment by 3941** [comment]
Wait... You are still updating the Chrome version, are you?
Last update January 2022 🤔

**Comment by vatara** [comment]
The change has been submitted, looks like the update is still pending review on the Chrome store. It should update within a day or so.

**Comment by 3941** [comment]
Awesome. I was just concerned by the version number. Looked like maybe you had skipped one on Chrome.
Maybe that update was only relevant for FIrefox!
All good, anyway, and thank you very much, again :)
